flink-tests/src/test/java/org/apache/flink/test/misc/SuccessAfterNetworkBuffersFailureITCase.java

```diff
@@ -59,7 +59,7 @@ public void testSuccessfulProgramAfterFailure() {
 			config.setInteger(ConfigConstants.LOCAL_NUMBER_TASK_MANAGER, 2);
 			config.setLong(TaskManagerOptions.MANAGED_MEMORY_SIZE, 80L);
 			config.setInteger(ConfigConstants.TASK_MANAGER_NUM_TASK_SLOTS, 8);
-			config.setInteger(TaskManagerOptions.NETWORK_NUM_BUFFERS, 800);
+			config.setInteger(TaskManagerOptions.NETWORK_NUM_BUFFERS, 1024);
```

Comment:
Is that also the reason here? I see that otherwise we get into `Insufficient number of network buffers` but it does not look as if it was configured as tightly...
(just want to rule out some memory leak with the new code)

Reply:
yes, the same reason as above

Reply:
Do you, by any chance, know why 800 worked and is not enough here anymore? I mean, why/how does this test need 800 buffers?